vere/http.c

```diff
@@ -2333,28 +2360,27 @@ _proxy_ward_start(u3_pcon* con_u, u3_noun sip)
 
     rev_u->por_s = ntohs(add_u.sin_port);
 
+#if 0
     {
-      c3_w* non_w = c3_malloc(64);
-
-      c3_rand(non_w);
-
-      u3_noun non = u3i_words(16, non_w);
-      c3_w len_w = u3r_met(3, non);
-
-      rev_u->non_u = uv_buf_init((c3_c*)non_w, len_w);
-
-      u3z(non);
+      u3_noun who = u3dc("scot", 'p', u3k(sip));
+      c3_c* who_c = u3r_string(who);
+      fprintf(stderr, "\r\nward for %s started on %u\r\n", who_c, rev_u->por_s);
```

Comment:
This actually seems like a great log message to have in production, as long as it doesn't spam the console. If only we had a syslog emitter or something like that ...

To be clear, I'm not suggesting any change to this right now.

Reply:
We open up way too many of these. I'd like to switch these proxy streams to be persistent and multiplex connections, using something like `newt.c` from cc-release. I think that would be simpler, cleaner, and more performant. Goals...